src/app/components/create-new-device/create-new-device.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CreateNewDeviceComponent } from './create-new-device.component';
+
+describe('CreateNewDeviceComponent', () => {
+  let component: CreateNewDeviceComponent;
+  let fixture: ComponentFixture<CreateNewDeviceComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CreateNewDeviceComponent ]
```

Comment:
if testing is something we still care about, this isn't going to work. This file will need things like the no errors schema and imports/providers for the classes it uses. login.component.spec.ts is a good example of what you need to do to get one of these testing files to work.